src/app/core/services/fss-search-filter.service.ts

```diff
@@ -24,7 +24,7 @@ export class FssSearchFilterService {
       var currentSearchRowJoinOperator = fssSearchRow.selectedJoinOperator;
       var currentSearchRowField = fssSearchRow.selectedField.replace(/'/g, "''");
       var currentSearchRowOperator = fssSearchRow.selectedOperator;
-      var currentSearchRowValue = fssSearchRow.value.replace(/'/g, "''");
+      var currentSearchRowValue = (fssSearchRow.value + "").replace(/'/g, "''");
```

Comment:
This change looks odd to me, what is the reason for it? I ask because of the following javascript behaviour:
> null 
= null
> null + "" 
= 'null'
> undefined 
= undefined 
> undefined + "" 
= 'undefined'
(the above is from chrome javascript console)

Could you please confirm this is the intended/expected behaviour?

Reply:
Hi Chris, its a wrong fix .. working on this